content/en/docs/components/serving/kfserving.md

```diff
@@ -88,7 +88,7 @@ Knative Serving (v0.11.2 +) and Istio (v1.1.7+), Cert Manager(v0.12.0+) should b
 Please see more details for installing the prerequisites [here](https://github.com/kubeflow/kfserving#prerequisites)
```

Comment:
Update: 

---
Line 88:

- Please add a space/blank line separators after each `## Section name` across the guide, if you have time
- For a11y and clarity, instead of `See [here]...` maybe we can use `refer to [README section]...`
- Add a full stop

```suggestion

For installing KFServing prerequisites, refer to the [README section](https://github.com/kubeflow/kfserving#prerequisites).
```

---

Line 87:

- Comma placement

```diff
- Knative Serving (v0.11.2 +) and Istio (v1.1.7+), Cert Manager(v0.12.0+) should be available on your Kubernetes cluster.
+ Knative Serving (v0.11.2 +), Istio (v1.1.7+), and Cert Manager(v0.12.0+) should be available on your Kubernetes cluster.
```

Reply:
Update:

Lines 81-83:

- Add full stops, if you have time

Reply:
Excellent suggestions :heavy_check_mark: